docs/writing-rules/rule-syntax.md

```diff
@@ -187,6 +187,29 @@ Comparison expressions support simple arithmetic as well as composition with [bo
 
 Building off of the previous example this will still catch code like `set_port(80)` but will no longer catch `set_port(443)` or `set_port(8080)`.
 
+The `comparison` key accepts Python expression using:
+
+- Boolean, string, integer, and float literals.
+- Boolean operators `not`, `or`, and `and`.
+- Arithmetic operators `+`, `-`, `*`, `/`, and `%`.
+- Comparison operators `==`, `!=`, `<`, `<=`, `>`, and `>=`.
+- Function `int()` to convert strings into integers.
+- Function `str()` to convert numbers into strings.
+- Lists and the `in` infix operator.
+- Function `re.match()` to match a regular expression (without the optional `flags` argument).
+
+And, of course, you can use Semgrep metavariables like `$MVAR`, which are evaluated as follows:
```

Comment:
- @IagoAbal do we have $MVAR documented somewhere? I found a few mentions in release notes, but we do not have anything more. This brings me to another question: Do you think that it makes sense to create an **exhaustive table for all metavariables** (like $FUNC, $ARG) we support or what metavariables to use is obvious for developers? (Can I find all of these metavariables somewhere in our source code to figure it out myself?)
- Please, can you make an example in Playground for use of  `$MVAR` as described below?

```suggestion
You can use Semgrep metavariables such as `$MVAR`, which Semgrep evaluates as follows:
```

Reply:
@IagoAbal crossed the text above, understood, metavariables are similar to variables in bash (with $ sign) - I can use anything I want ($X), it is to make the rule filled with meaningful text for people reading the rule, but it does not affect the behavior of Semgrep whether we use $X or $FUNC.

Reply:
@adamkvitek As discussed - keep $MVAR. All Iago has to do is accept the suggestions. I don't see anything that changes the meaning too much.

Reply:
Yes, that is what I tried to achieve, so @IagoAbal can only apply suggestions.